Global/JetBrains.gitignore

```diff
@@ -10,6 +10,7 @@
 
 # Generated files
 .idea/**/contentModel.xml
+.idea/**/projectSettingsUpdater.xml
```

Comment:
I'd add a comment that this is a Rider specific file, rather than JetBrains wide.

You could also make the pattern a little more Rider specific with `.idea/*/.idea/projectSettingsUpdater.xml`, as Rider has a nested `.idea` folder to handle multiple solution files in the same folder. (.NET solutions are file based, and other JetBrains IDEs' projects are folder based)